components/autofill/sql/create_shared_schema.sql

```diff
@@ -46,7 +46,12 @@ CREATE TABLE IF NOT EXISTS addresses_tombstones (
 CREATE TABLE IF NOT EXISTS credit_cards_data (
     guid                TEXT NOT NULL PRIMARY KEY CHECK(length(guid) != 0),
     cc_name             TEXT NOT NULL, -- full name
-    cc_number           TEXT NOT NULL, -- TODO: consider storing this field as a hash
+    -- Encrypted card number. While this length is variable it must be > 19 chars,
+    -- so CHECK that to avoid somehow storing a non-encrypted one.
+    -- XXX - is this a good idea?
+    cc_number_enc       TEXT NOT NULL CHECK(length(cc_number_enc) > 20),
```

Comment:
I was wondering if rust code shouldn't be handling this with a custom error of some kind but I guess this is just a fail-safe as we expect the consumer to handle validating the number.